bin/before_script.sh

```diff
@@ -2,45 +2,61 @@
 
 # before_script
 export PATH="$HOME/.composer/vendor/bin:$PATH"
-export REPOSITORY_NAME=$(find $TRAVIS_BUILD_URL -mindepth 1 -maxdepth 1 -name "*.info" -type f -printf '%f\n' | cut -f1 -d".")
+export REPOSITORY_NAME=$(basename $TRAVIS_BUILD_DIR)
+
 # download linky_clicky and copy over related tests and required files
 if [ ! -z "$CLICKY_BRANCH" ]; then CLICKY_BRANCH="-b $CLICKY_BRANCH"; fi
+echo "git clone --depth 1 $CLICKY_BRANCH https://github.com/SU-SWS/linky_clicky.git $HOME/linky_clicky"
 git clone --depth 1 $CLICKY_BRANCH https://github.com/SU-SWS/linky_clicky.git $HOME/linky_clicky
 mkdir -p $HOME/stanford_travisci_scripts/includes/config
 mkdir $HOME/stanford_travisci_scripts/includes/extensions
-ls $HOME/stanford_travisci_scripts/features
+cp $HOME/linky_clicky/includes/bootstrap/* $HOME/stanford_travisci_scripts/features/bootstrap/.
+cp $HOME/linky_clicky/includes/config/default.yml $HOME/stanford_travisci_scripts/includes/config/.
+cp $HOME/linky_clicky/includes/extensions/drupal.extension.yml $HOME/stanford_travisci_scripts/includes/extensions/.
+cp $HOME/linky_clicky/includes/extensions/mink.extension.yml $HOME/stanford_travisci_scripts/includes/extensions/.
 
-# copy over product tests
-if [ "$REPOSITORY_NAME" == "stanford-jumpstart-deployer" ]; then
+# copy over production product tests
+function copy_product_tests {
   declare -A PRODUCTS_LIST=(
     ["jumpstart-academic"]="jsa"
+    ["jumpstart-engineering"]="jse"
     ["jumpstart-plus"]="jsplus"
     ["jumpstart"]="jsv"
     ["jumpstart-lab"]="jsl"
   )
-  SUFFIX="${PRODUCTS_LIST[$PRODUCT_NAME]}"
-  echo "Suffix: $SUFFIX"
-  echo "cp -r $HOME/linky_clicky/products/$SUFFIX/features $HOME/stanford_travisci_scripts/features/$REPOSITORY_NAME"
-  cp -r $HOME/linky_clicky/products/$SUFFIX/features $HOME/stanford_travisci_scripts/features/$REPOSITORY_NAME
-fi
+  ACRONYM="${PRODUCTS_LIST[$PRODUCT_NAME]}"
```

Comment:
Let me know if the lines below adequately check whether the variable has a value.